napari/layers/layer_event_handler.py

```diff
@@ -0,0 +1,42 @@
+import logging
+
+
+from ..utils.event_handler import EventHandler
+
+
+logger = logging.getLogger(__name__)
+
+
+class LayerEventHandler(EventHandler):
+    """
+    Event handler for layer specific events. Receives change events made from
+    the data, controls, or visual interface and updates all associated
+    components.
+
+    Ex. ImageLayer components = [qt_image.py, vispy_image.py, image.py]
+    """
+
+    def __init__(self, component=None):
+        super().__init__(component)
+
+    def on_change(self, event=None):
+        """
+        Process changes made from any interface
+        """
```

Comment:
```suggestion
        """Process changes made from any interface."""
```

([PEP257](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings))

Reply:
done